tests/wpt/servowpt.py

```diff
@@ -154,10 +154,23 @@ def run_tests(**kwargs):
 
     # Filter intermittents if that was specified on the command-line.
     if handler.unexpected_results and filter_intermittents_output:
-        all_filtered = filter_intermittents(
-            handler.unexpected_results,
-            filter_intermittents_output
-        )
+        unexpected_results = list(handler.unexpected_results)
+
+        print(80 * "=")
+        print(f"Rerunning {len(unexpected_results)} tests "
+              "with unexpected results to detect flaky tests.")
+        unexpected_results_tests = [result.path for result in unexpected_results]
+        kwargs["test_list"] = unexpected_results_tests
+        kwargs["include"] = unexpected_results_tests
+        kwargs["pause_after_test"] = False
+        wptrunner.run_tests(**kwargs)
+
+        stable_tests = [result.path for result in handler.unexpected_results]
+        for result in unexpected_results:
+            result.flaky = result.path not in stable_tests
```

Comment:
suggest commenting

```python
        # Mark any tests that yielded expected results when rerun as flaky, but
        # otherwise disregard the results of the second run.
```

Reply:
I've added a similar comment which also points to a future behavior which might be good as well.

```
+        # Use the second run to mark tests from the first run as flaky, but
+        # discard the results otherwise.
+        # TODO: It might be a good idea to send the new results to the
+        # dashboard if they were also unexpected.
```